src/runtime_src/core/pcie/tools/xbutil/xbutil.cpp

```diff
@@ -1750,6 +1763,33 @@ int xcldev::xclReset(int argc, char *argv[])
         return -EINVAL;
     }
 
+    std::string vbnv, errmsg;
+    auto dev = pcidev::get_dev(index);
+    dev->sysfs_get( "rom", "VBNV", errmsg, vbnv );
+    if (!errmsg.empty()) {
+        std::cout << errmsg << std::endl;
+        return -EINVAL;
+    }
+    if (!all && vbnv.find("_u30_") != std::string::npos) {
+        std::stringstream dbdf;
```

Comment:
Not clear why you need it instead of directly outputing the messages...

Reply:
I am constructing a dbdf string, which is part of the shell cmd. that is not for outputing